README.md

```diff
@@ -55,7 +55,15 @@ To run the app and have hot-reloads for development:
 
 - Running end to end tests:
 
-      yarn run test:e2e
+    yarn run test:e2e
+
+    You will need to make the following changes to get it working:
+        * if you don't already have one create `.env.production.local` in the root folder as a copy of `.env.production`
+        * change `VUE_APP_TRACTION_BASE_URL=REPLACE_VUE_APP_TRACTION_BASE_URL to VUE_APP_TRACTION_BASE_URL=http://traction`
+        * change `VUE_APP_SAMPLEEXTRACTION_BASE_URL=REPLACE_VUE_APP_SAMPLEEXTRACTION_BASE_URL to VUE_APP_TRACTION_BASE_URL=http://samples-extraction`
```

Comment:
'to VUE_APP_TRACTION_BASE_URL=http://samples-extraction' should be VUE_APP_SAMPLEEXTRACTION_BASE_URL=http://samples-extraction ?